crates/sui-core/src/authority_client.rs

```diff
@@ -89,6 +92,7 @@ impl NetworkAuthorityClient {
     pub fn new(channel: tonic::transport::Channel) -> Self {
         Self {
             client: ValidatorClient::new(channel),
+            metrics: NetworkAuthorityClientMetrics::new(&prometheus::Registry::new()),
```

Comment:
I'm not sure using a new Registry the metrics will be collected? We probably need to pass the centralized registry all the way from e.g. [here](https://github.com/MystenLabs/sui/blob/main/crates/sui-node/src/lib.rs#L187)

Reply:
Is there a local way to test for these things (ie will it get collected / reported)? Right now the only way is to push to production.